integration-guides/mixtelematics.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: MiX Telematics Australia
+---
+
+To connect your Mix Telematics device integration with PathStack, you will need to obtain your Organisation Group ID, Client ID, Username and Password for accessing the API. The subsequent procedures will guide you through the process. 
+
+<Steps>
+
+  <Step title="Log into your MixFleet account">
+To access your username, password and group Id, you will need to log into your MixFleet manager portal.  The password and username that you use to access the MixFleet portal are the same credentials that you will need for the API connection. 
+  <img src="https://365scotney-my.sharepoint.com/:i:/r/personal/sameer_nanduri_inloop_com_au/Documents/Pathstack%20Images/Mix_1.png?csf=1&web=1&e=W7ABsh" />
+</Step>
+
+  <Step title="Find your Organisation ID">
+   Whilst logged into the portal you will need to obtain the Organisation Group ID. This ID will be used as part of the source settings API credentials.
+
+   Once logged into Mixfleet, click on the Asset icon and then click on any asset within your fleet. 
+     <img src="https://365scotney-my.sharepoint.com/:i:/r/personal/sameer_nanduri_inloop_com_au/Documents/Pathstack%20Images/Mix_2.png?csf=1&web=1&e=W7ABsh" />
+    when you have selected an asset, you will be able to find the organisation id in the webpage url (as shown in picture below). 
+    <img src="https://365scotney-my.sharepoint.com/:i:/r/personal/sameer_nanduri_inloop_com_au/Documents/Pathstack%20Images/Mix_3.png?csf=1&web=1&e=W7ABsh" />
+
+  </Step>
+  <Step title="Register for a Client ID and Client Secret via Mix Telematics ">
+  To be set up with a Client ID and Client Secret, you will need to contact your regional sales office or your telematics sales representative and request to use the Mix Integrate API platform for the purpose of connecting to PathStack. 
+Your telematics representative will need the following information: 
+
+<Steps>
+<Step title = "Software application name (Name of the tracking solution): PathStack">
```

Comment:
name shouldn't be capitalised

Reply:
don't have content after the colon in the title, add it in between the step tags. Do same for all 5 steps